src/main/java/seedu/address/logic/parser/ScheduleAllCommandParser.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.ScheduleAllCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ScheduleAllCommand object.
+ */
+public class ScheduleAllCommandParser implements Parser<ScheduleAllCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ScheduleAllCommand
+     * and returns a ScheduleAllCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ScheduleAllCommand parse(String args) throws ParseException {
+        requireNonNull(args);
```

Comment:
I realised do we need parser for scheduleall? Args in this case will be "" requireNonNull might not be useful here

Reply:
Yep I agree with this comment. Consider looking at how addressBookParser handles the list command since the command format is similar where it is just the command word